### DISCUSSION TITLE
Auto-block/one-click way to block fake accounts

### DISCUSSION THREAD

**Discussion by Shohreh** [insin/control-panel-for-twitter#450] [Ideas]
Hello,

At least once a day, I get likes from fake accounts, which are invariably scantily clad "women" with "0 following 0/single-digit followers".

I suggest adding a way to either auto-block those fake accounts when they like a tweet or follow one's account, or provide a one-click solution to perform a manual block.

Cheers,

![image](https://github.com/insin/control-panel-for-twitter/assets/1541861/05c3cfe6-6c55-4d47-9420-a1d5b8233b7e)

**Comment by Damrak1** [comment]
I support that idea, I receive many likes from bots every day. It would be great to have a few options added to 

https://twitter.com/settings/notifications/advanced_filters

to block or mute a user based on their profile:

- a word (like 'follow')
- the month the account was created in (always recent)
- the number of tweets (always less than 10).
- the number of followers (very few)

(_The example is in Dutch_)

![bot2](https://github.com/insin/control-panel-for-twitter/assets/103885608/be017a66-30d1-4caf-bdd1-8dbdad66c6d7)

**Comment by Shohreh** [comment]
Pattern:
1. Always has a profile picture ('f course)
2. Joined very recently ("September 2023")
3. "0 Following 0 Followers" (or single-digit)
